client/src/components/Poster.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+Poster.defaultProps = {
+  path: 'https://via.placeholder.com/300x450?text=POSTER',
+  alt: 'Poster of Series',
+}
```

Comment:
Many tend to use the default params syntax instead of `defaultProps`. Feel free to use what fit your needs best.